app/uk/gov/hmrc/soletraderidentificationfrontend/testonly/views/test_create_individual_journey.scala.html

```diff
@@ -84,6 +85,12 @@
             classes = "govuk-input--width-20"
         )
 
+        @inputHidden(
+            id = "businessVerificationCheck",
+            name = "businessVerificationCheck",
+            value = form("businessVerificationCheck").value.getOrElse("true")
+        )
+
```

Comment:
Do we need this on the individual creation page? Why is it hidden?

Reply:
The business verification deactivation is not relevant to the individual journey so I don't think we need the checkbox. However, to keep the form happy I thought it a good idea to submit something. Further, if we submit nothing it is likely the business verification flag will be set to "false". As the default value is actually "true" I think it is better to use "true".